components/job-orchestration/job_orchestration/scheduler/query/query_scheduler.py

```
@@ -982,7 +982,7 @@ async def handle_finished_stream_extraction_job(
         if not QueryTaskStatus.SUCCEEDED == task_result.status:
             logger.error(
                 f"Extraction task job-{job_id}-task-{task_id} failed. "
-                f"Check {task_result.error_log_path} for details."
+                "Check container logs for details."
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)